static/app/views/dashboardsV2/detail.tsx

```diff
@@ -243,6 +247,7 @@ class DashboardDetail extends Component<Props, State> {
       this.setState({
         dashboardState: DashboardState.VIEW,
         modifiedDashboard: null,
+        layout: getDashboardLayout(organization.id, dashboard.id),
```

Comment:
Should I feature flag this `getDashboardLayout` call? Same with the one in the initial state.

Reply:
Probably ok to leave as is. Looks like only `GridLayoutDashboard` uses it and that's already protected by a feature flag